src/Reflection/ReflectionMethod.php

```diff
@@ -26,20 +27,16 @@ class ReflectionMethod extends AbstractFunction
 
     /**
      * Parent class reflection
-     * @var ReflectionClass|\ReflectionClass
+     * @var ReflectionClass
      */
     protected $classReflection;
 
-    /**
-     * Constructor
-     *
-     * @param ReflectionClass $class
-     * @param \ReflectionMethod $r
-     * @param string $namespace
-     * @param array $argv
-     */
-    public function __construct(ReflectionClass $class, \ReflectionMethod $r, $namespace = null, $argv = [])
-    {
+    public function __construct(
+        ReflectionClass $class,
+        \ReflectionMethod $r,
+        ?string $namespace = null,
+        ?array $argv = null
```

Comment:
signature change, probably we want to keep just array, without nulls

Reply:
That is correct. Fixed.